doc/SONIC_Test_Ingress_Discards_HLD.md

```diff
@@ -994,3 +997,35 @@ Packet1 to send
 - Get L2 drop counter
 - Verify L2 drop counter is not incremented
 - Enable back egress interface on DUT which is linked with neighbouring device
+
+#### Test case #22
+##### Test objective
+Verify ingress Priority Group drop packets counter
+
+Packet to send
+```
+...
+###[ IP ]###
+  version= 4
+  tos= [(lossless_priority << 2) | ECN]
```

Comment:
Can we get DSCP to TC mapping from config db and calculate tos value based on that?

Reply:
Yes, probably like there 
https://github.com/ayurkiv-nvda/sonic-mgmt/blob/master/tests/common/fixtures/pfc_asym.py#L364